config/helm/aws-node-termination-handler/values.yaml

```diff
@@ -4,123 +4,129 @@
 
 image:
```

Comment:
That wouldn't work, the chart defaults need to be in _values.yaml_ and for the tooling to work correctly all values called by name should be in that.

We could add _example-values-queue.yaml_, _example-values-imds-linux.yaml_ & _example-values-imds-windows.yaml_ to the chart (and put them in _.helmignore_) to show the use. These could then be used by the lint testing rather than the contrived values trying to set everything.

Alternatively we could split out the IMDS functionality into a new chart and simplify the whole thing?

Reply:
I was thinking more of the `example-values-queue.yaml` style thing.  At this point publishing 2 helm charts for IMDS and Queue-Processor would be awkward I think.  I think we can solve this drift in our v2 that we'll start working on very soon.

Reply:
I agree that this would be the best approach, I'll make the changes.

Reply:
Done.